pkg/config/config.go

```diff
@@ -2035,13 +2035,13 @@ func getValidHostAliasesWithConfig(config Config) []string {
 //
 // This is composed of DD_TAGS and DD_EXTRA_TAGS, with DD_DOGSTATSD_TAGS included
 // if includeDogstatsd is true.
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
`GetGlobalConfiguredTags` and `GetConfiguredTags` have the same description now. `GetConfiguredTags` description could become 
```
// GetConfiguredTags returns list of tags from a configuration, based on tags and extra_tags, with dogstatsd_tags if includeDogdstatsd is true. 
```